src/Resources/views/Collector/theme.html.twig

```diff
@@ -19,9 +19,16 @@
                 <b>Used themes</b>
                 <span class="sf-toolbar-status">{{ collector.usedThemes|length }}</span>
             </div>
+
+            {% for theme in collector.themes %}
+                <div class="sf-toolbar-info-piece">
+                    <b>{{ theme }}</b>
+                    <span>(<a href="?{{ constant('Sylius\\Bundle\\ThemeBundle\\Context\\FakeTheme\\FakeThemeNameProviderInterface::PARAMETER_NAME') }}={{ theme.name }}">change</a>)</span>
```

Comment:
you should handle existing query params ? Or at least handle ? vs &

Reply:
I relied on what is done for channel switcher. But indeed, it would make sense to handle those cases. I will try to fix this

Reply:
Having second thoughts on that. I don't think it is needed to handle that, since first, there is no route in Sylius out of the box with query params.
And second, even if there is, switching theme is stored in session, and then you can go back to that page after switching instead of handling that.
And third, right now it works even if there is already a query param. This one is simply overriding any order. So you could just go back to that page afterward imo.

WDYT ?